### ISSUE TITLE
Feature request: distcc support

### ISSUE THREAD

**Issue by uzsolt** [freebsd/poudriere#408]
I think it would be a good thing to support distcc.

**Comment by bdrewery** [comment]
Agreed, I want to add it to the base system first.

**Comment by voltagex** [comment]
Ping on this issue - it'd be very helpful when building packages for much slower systems.

**Comment by bapt** [comment]
too be honest it might just be impossible due to the fact that it breaks some of the design goals of poudriere: the build is happening in jail with NO network at all, hence running distcc in it would be a problem.

**Comment by darkfiberiru** [comment]
I agree that adding support would be additional work to more parts of poudriere and need to not effect poudriere operation when not using the feature.  But I don't see how that precludes this as a feature. I say this as just a system administrator and user of poudrirere. I guess what I'm trying to ask @bapt or anyone else is what technical challenges or worries about breaking poudriere  do you have.

**Comment by gbeine** [comment]
Hi folks,
I'm running distcc with poudriere on a Raspberry Pi 2 and an AMD64 distcc server.
Works quite well with only minor changes.
I've forked poudriere and pushed these changes on my fork repository:
https://github.com/gbeine/poudriere/tree/DISTCC_FIX
So long...

**Comment by voltagex** [comment]
@gbeine can you rebase your fork on top of the current code? What did you have to do to install your version of poudriere on the system? (I currently use the version from pkg)

**Comment by gbeine** [comment]
I'll try to do a rebase, but I did not use it since a while.
For poudriere I did not need anything on the base system, but of course distcc clients in every jail.

**Comment by GitHubinat0r** [comment]
@gbein did you get anywhere with this? I have a use case where distcc would be helpful in combination with Poudriere.

**Comment by debdrup** [comment]
Anyone interested in this might wanna look at https://github.com/freebsd/poudriere/projects/2
